docroot/themes/humsci/su_humsci_theme/patterns/molecules/horizontal-card/scss/horizontal-card.scss

```diff
@@ -97,4 +97,15 @@
       }
     }
   }
+
+  // Ensures desired layout when used on page using the three column with image layout
+  .three-column-w-image & {
```

Comment:
Why not just target the heading inside this pattern. eliminating the cross reference.

Reply:
Fixed